integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -103,6 +103,17 @@ public interface TestConstants {
   public static final String WEBLOGIC_IMAGE_TAG = getNonEmptySystemProperty("wko.it.weblogic.image.tag", 
        WEBLOGIC_IMAGE_TAG_DEFAULT);
 
+  public static final String BASE_WEBLOGIC_IMAGE =
+      BASE_IMAGES_REPO + "/" + getNonEmptySystemProperty("wko.it.weblogic.image.name", WEBLOGIC_IMAGE_NAME_DEFAULT);
+  public static final String TEST_WEBLOGIC_IMAGE =
+      TEST_IMAGES_REPO + "/weblogick8s/test-images/weblogic";
+
+  public static final String BASE_WEBLOGIC_IMAGE_NAME = BASE_WEBLOGIC_IMAGE + ":" + WEBLOGIC_IMAGE_TAG;
```

Comment:
where are we using BASE_WEBLOGIC_IMAGE_NAME?

Reply:
I introduced this new variable to match with corresponding TEST_WEBLOGIC_IMAGE_NAME. I will remove it. It is not being used in any class.